tauri-app/src/lib/services/executeWalletConnectOrder.ts

```
@@ -0,0 +1,49 @@
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  formatEthersTransactionErrorFn: (error: any) => string,
+  successToastFn: (message: string) => void,
+  errorToastFn: (message: string) => void,
+): Promise<void> {
+  if (!deployment) throw Error('Select a deployment to add order');
+  if (isEmpty(deployment.order?.orderbook) || isEmpty(deployment.order.orderbook?.address))
+    throw Error('No orderbook associated with scenario');
+
+  try {
+    const calldata = await orderAddCalldataFn(dotrainText, deployment);
+    const tx = await ethersExecuteFn(calldata, deployment.order.orderbook.address!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)